core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -119,6 +121,12 @@ oppia.controller('StateResponses', [
       }
     };
 
+    $scope.isCorrectnessFeedbackEnabled = function() {
+      $scope.temp =
```

Comment:
Why define $scope.temp at all? The name isn't meaningful, and in fact you can probably get rid of it altogether by just doing

    return (
      explorationCorrectnessFeedbackService.isCorrectnessFeedbackEnabled());

Reply:
It still exceeds the maximum line length of 80. Please help me to correct this error.

Reply:
Even if you break after the first '('?

Reply:
Yep! This whole line exceeds the limit (the length is 85)
"explorationCorrectnessFeedbackService.isCorrectnessFeedbackEnabled());"

![max_limit](https://user-images.githubusercontent.com/23234392/34431699-4e5962a6-ec97-11e7-93ff-4a7d4ad9704c.png)

Reply:
Oh, that's AnswerGroupDirective.js. The file I'm commenting on is StateResponses.js. That's why I was confused.

I would rename explorationCorrectnessFeedbackService.isCorrectnessFeedbackEnabled() to explorationCorrectnessFeedbackService.isEnabled().

Reply:
Okay! So shall I change the function to "isEnabled()" in EditorService.js ?

Reply:
Yes, that's fine. It's pretty clear from the name of the service what's being enabled.

Reply:
Done!